REVIEWING.md

```diff
@@ -0,0 +1,108 @@
+
+## How are reviews performed
+
+The Podman project aims to ensure that all PRs are reviewed by at least 2 people prior to merge, at least one of which must be a repository Maintainer.
+There are some exceptions to this: Updates to libraries (including Go vendor updates) that pass CI cleanly and require no code changes may be merged by a maintainer without further review.
+
+All code merged must pass CI.
+
+## What should you review?
+
+We encourage review of all PRs, even those not in the area of expertise of the reviewer.
+Even if you are not fully familiar with the code in question, you can still notice basic mistakes.
+Feel free to ask questions about how areas of the code work to help familiarize yourself during reviews.
+If you finish a review and do not feel like you adequately understood the code to approve it for merge, tag an expert in that area of the code to perform a further review.
+
+Timely PR reviews are important - contributors can become discouraged if a PR is neglected by maintainers.
+All Maintainers and Reviewers should try to review new pull requests in their repositories once a day to ensure this.
+When you review a PR with failing tests, please check to see if those tests failed due to known flakes.
+If so, please restart the failed tests.
+Many repositories in the Podman project can only have their tests restarted by project members, not the submitter, so regular attention to test failures by reviewers is important.
+
+## Things to Check
+
+### Breaking Changes
+
+The Podman project aims to present a stable API for its users.
+Breaking changes to the project's Command Line Interfaces or public APIs (the Podman REST API and its associated bindings) must only be made in approved breaking change releases - Podman 6.0, 7.0, etc.
+Breaking changes can include renaming an option without retaining the original name as an alias, removing an option entirely, or changing how an option works in a way that does not ensure backwards compatibility.
+
+Periods when it is acceptable to merge breaking changes will be widely announced.
+If it is not one of those periods, reviewers should be on the lookout for breaking changes.
+PRs with breaking changes should not be merged.
+Please guide the contributor in how to make the change in a non-breaking fashion.
+If this is not possible, deferring the PR to the next breaking change window or closing it entirely is appropriate.
+
+### Commit Messages and Hygiene
+
+Good commit messages are essential for understanding why a change was made in the future.
+Reviewers should check each commit of a PR to ensure that it has an appropriate commit message which fully and accurately explains the change and why it is being made.
+This should remain true after changes to the PR due to code review, so please re-review commit messages before merging code, to ensure they are still accurate.
```

Comment:
Commit messages must have `Fixes: #xxxx` with the right issue number if they claim to fix said issue.